Formula/s/sile.rb

```diff
@@ -193,27 +193,12 @@ def install
       --disable-silent-rules
       --with-system-luarocks
       --with-system-lua-sources
-      --disable-embeded-resources
     ]
 
-    # Upstream bug https://github.com/sile-typesetter/sile/issues/2078 triggers
-    # a useless automake cycle when building from the source tarball. This
-    # argument avoids needing the dependency by just making it a noop. Remove
-    # on the next release.
-    configure_args += %w[AUTOMAKE=:]
-
     system "./bootstrap.sh" if build.head?
     system "./configure", *configure_args, *std_configure_args
     system "make"
     system "make", "install"
-
-    env = {
-      LUA_PATH:  "#{ENV["LUA_PATH"]};;",
-      LUA_CPATH: "#{ENV["LUA_CPATH"]};;",
```

Comment:
We should probably still append `;;` to these paths in their definition above so that the system paths are available.

Reply:
Like that?